test-scenarios/doubly-indirect-test.ts

```diff
@@ -51,8 +51,25 @@ function makeAddon() {
   addon.addDependency('some-lib', {
     files: {
       'index.js': `
+
+    class ClassWithStaticBlockAndPrivateProperties {
```

Comment:
Not sure if it is the best place to test that. Please let me know if there is a better place. Maybe a new test file?